AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoModel.swift

```diff
@@ -205,13 +206,18 @@ class VideoModel: NSObject {
             } else if lhsIsActiveSpeaker && !rhsIsActiveSpeaker {
                 return ComparisonResult.orderedAscending
```

Comment:
why is  `alreadySorted = false` not in this case?

Reply:
Because this would indicate ascending order which mean already sorted.